src/main/resources/db/migration/all/20240515143555__add_missing_indexes_to_fks.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX ON domain_event(assessment_id);
```

Comment:
should this be `domain_events(..)`?

Reply:
that would explain all the test failures!